specification/web/resource-manager/Microsoft.Web/2016-09-01/AppServicePlans.json

```diff
@@ -3214,7 +3433,8 @@
               "readOnly": true
             },
             "certBlob": {
-              "description": "A certificate file (.cer) blob containing the public key of the private key used to authenticate a \n Point-To-Site VPN connection.",
+              "format": "byte",
+              "description": "A certificate file (.cer) blob containing the public key of the private key used to authenticate a \nPoint-To-Site VPN connection.",
```

Comment:
A potential SDK breaking changes.

Reply:
Client validation for (uuid, byte ) will cause SDK breaks, but is acceptable and actually a good change.